app/models/wizard/steps/base.rb

```diff
@@ -0,0 +1,19 @@
+module Wizard
+  module Steps
+    class Base
+      include ActiveModel::Model
+
+      attr_reader :session,
+                  :attributes
+
+      def initialize(session, attributes = {})
```

Comment:
Could we make the session an active model attribute rather than overriding the constructor?

```ruby
question = Wizard::ImportDate.new(session: session, **attributes)
question.import_date => '2020-01-01'
```

I think overriding the constructor is breaking the normal getter/setter api of ActiveModel::Model which assigns in the constructor all of the values to instance variables or something.

Also happy that the answer might be in a later PR since it would probably break some tests/delay things to do that

Reply:
Actually, all we need to do is call super passing the attributes

Reply:
```suggestion
      def initialize(session, attributes = {})
        super attributes
```

Reply:
Let's chat to make sure I understand.

Reply:
Btw I like the format you proposed `2020-01-01` better for the date. Will change to use that.

Reply:
I've refactored this.